### ISSUE TITLE
Investigate npm/node.js-free possibility

### ISSUE THREAD

**Issue by mkotsbak** [chandu0101/sri#14]
I see that the project is depending heavily on npm packages and dependencies as well as sbt. It would be easier to use if only sbt was required to build and use the library.

Do you think that is possible? All dependencies from npm/bower is possible to find/import to webjars.org and get from sbt.

**Comment by chandu0101** [comment]
for mobile examples we must use npm/node.js

for web examples we can replace with sbt but from my previous experience all js components/frameworks don't publish to webjars and npm.webjars doesn't work out of the box in few cases. i personally prefer npm/node.js(easy to learn and we must need this when developing mobile apps)  for js deps. That being said i am happy to take PR's for template projects(sri-web-sbt ... )  based on sbt,

**Comment by mkotsbak** [comment]
Yes, the infrastructure and scripts from react tools are probably useful. But we might get them in jars and connect them using sbt targets.

Yes, imports sometimes fails, but they fix the issues quickly: https://github.com/webjars/webjars/issues/1246 and https://github.com/webjars/webjars/issues/1247

**Comment by mkotsbak** [comment]
And upstream FB has fixed this issue for next version: https://github.com/facebook/react/issues/5461

**Comment by chandu0101** [comment]
> Yes, the infrastructure and scripts from react tools are probably useful. But we might get them in jars and connect them using sbt targets.

I must admit that i'am sbt noob , not sure whether its an easy task or not, especially for react-native tool chain and relay-babel-plugin!. But its good to have templates with sbt only, looking forward to u r findings ;)

**Comment by onilton** [comment]
@mkotsbak I've update the web-template to remove npm need :). Take a look and tell me what you think:

https://github.com/onilton/sri-web-template
